src/logging.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+import * as winstonLogger from './Adapters/Logger/WinstonLogger';
+
+let logger = winstonLogger.logger;
+export function setLogger(aLogger) {
+  if (!aLogger) {
+    console.log('NO LOGGER SET!');
+  }
+  logger = aLogger;
+}
+
+function getLogger() {
+  return logger;
+}
+
+export function addTransport(transport) {
```

Comment:
just to make sure i am following, it isn't the winstonAdapter that handles the transports, its WinstonLogger, right?